app.js

```diff
@@ -32,14 +25,16 @@ app.engine('handlebars', exphbs({defaultLayout: 'main'}));
 let redis;
 if (process.env.REDIS_URL) {
   const rtg = require('url').parse(process.env.REDIS_URL);
-  redis = require('redis').createClient(rtg.port, rtg.hostname);
+  redis = require('redis').createClient({ port: rtg.port, host: rtg.hostname, password: process.env.REDIS_PASSWORD });
+} else {
+  redis = require('redis').createClient();
```

Comment:
We'll have to add to the README instructions and setup stuff to make sure redis is installed